requirements/requirements_dev.txt

```diff
@@ -8,3 +8,4 @@ pytest-xdist
 pytest-cov
 pytest-django
 setuptools-scm
+psycopg
```

Comment:
By default `psycopg` uses pure python installation, which dynamically loads `libpq` and calls it via `ctypes`. If compatibility and availability of binary distributions is not an issue, I'd suggest considering using `psycopg[binary]` for development environment.

Please note though, that `psycopg[binary]` should not be used for production environments. For production environments the `psycopg[c]` is the preferred type of installation.

See https://www.psycopg.org/psycopg3/docs/basic/install.html for more details.